<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CandidatInterfaceClass</class>
 <widget class="QDialog" name="CandidatInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CandidatInterface</string>
  </property>
  <widget class="QLabel" name="label_prenom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Prénom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom_ville">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de la ville</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_num_rue">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro de la rue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_code_postal">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>140</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code postal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_province">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Province</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom_rue">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom de la rue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>NAS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_parti">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Parti politique </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenom">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nas">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_rue">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_numero_rue">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>40</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_nom_ville">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code_postal">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>130</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_province">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Bouton_ValiderCandidat">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>191</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider le nouveau candidat</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitrePageCandidat">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ajout d'un Candidat</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>185</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="liste_partis_politiques">
      <property name="text">
       <string>1. BLOC_QUEBECOIS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="liste_partis_politiques_2">
      <property name="text">
       <string>2. CONSERVATEURS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="liste_partis_politiques_3">
      <property name="text">
       <string>3. INDEPENDANT</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="liste_partis_politiques_4">
      <property name="text">
       <string>4. LIBERAL</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="liste_partis_politiques_5">
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="text">
       <string>5. NOUVEAU PARTI DEMOCRATE</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_date_naissance">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date de naissance</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>110</width>
     <height>27</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>23</hour>
     <minute>59</minute>
     <second>59</second>
     <year>1997</year>
     <month>1</month>
     <day>1</day>
    </datetime>
   </property>
   <property name="maximumDate">
    <date>
     <year>2037</year>
     <month>12</month>
     <day>31</day>
    </date>
   </property>
   <property name="minimumDate">
    <date>
     <year>1970</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>1997</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_partis_politiques">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>49</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>Bouton_ValiderCandidat</sender>
   <signal>clicked()</signal>
   <receiver>CandidatInterfaceClass</receiver>
   <slot>validerCandidat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>validerCandidat()</slot>
 </slots>
</ui>
